groups/alteryx-sandbox/variables.tf

```diff
@@ -220,3 +220,23 @@ variable "repository_name" {
   description = "The name of the repository in which we're operating"
   type        = string
 }
+
+# ------------------------------------------------------------------------------
+# Alteryx Server ALB Variables
+# ------------------------------------------------------------------------------
+
+variable "application_subnets" {
+  type        = list(string)
+  description = "The application subnets to be used"
+}
```

Comment:
These should be dynamically discovered via a `data` lookup based on a supplied subnet pattern.

Reply:
Have added a data lookup to do this